### ISSUE TITLE
Resgression output rather than classification

### ISSUE THREAD

**Issue by atinzad** [mdeff/cnn_graph#32]
Hi,

I was wondering if there is a code modification where the model is used to train for regression rather than classification. I assume that the loss function would be changed, as well as the last layer of the model.

Thank you

**Comment by mdeff** [comment]
Yep that should be a simple modification. The purpose of this repo is however limited to allow a reproduction of the paper. For new implementations I recommend you to look at [PyTorch Geometric](https://pytorch-geometric.readthedocs.io) and [Deep Graph Library](https://www.dgl.ai).
